fusor-ember-cli/app/routes/deployment.js

```diff
@@ -28,18 +28,20 @@ export default Ember.Route.extend(DeploymentRouteMixin, {
     var orgID = model.get('organization.id');
     var url = '/katello/api/v2/organizations/' + orgID;
     Ember.$.getJSON(url).then(function(results) {
-      if (Ember.isPresent(results.owner_details.upstreamConsumer)) {
-        controller.set('organizationUpstreamConsumerUUID', results.owner_details.upstreamConsumer.uuid);
-        controller.set('organizationUpstreamConsumerName', results.owner_details.upstreamConsumer.name);
-        // if no UUID for deployment, assign it from org UUID
-        if (Ember.isBlank(controller.get('model.upstream_consumer_uuid'))) {
-          controller.set('model.upstream_consumer_uuid', results.owner_details.upstreamConsumer.uuid);
-          controller.set('model.upstream_consumer_name', results.owner_details.upstreamConsumer.name);
+      Ember.run(function(){
```

Comment:
If I don't wrap this code in `Ember.run`, then I get this error.
```
Assertion Failed: You have turned on testing mode, which disabled the run-loop's autorun.
```
If it's included, then this error
```
Assertion Failed: You can only unload a record which is not inFlight.
```